.gitignore

```diff
@@ -15,3 +15,5 @@ Cargo.lock
 
 
 # End of https://www.gitignore.io/api/rust
+
+.idea
```

Comment:
Ignore rules for editors should be part of the global user ignore settings.

Reply:
good point, thanks i forgot that existed